lib/commanded/event_store/term_serializer.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Infra.TermSerializer do
+  def serialize(term) do
```

Comment:
No need to add this, there is already a test term serializer for testing purposes see: https://github.com/fabriziosestito/commanded-spear-adapter/blob/main/test/support/term_serializer.ex